course-backend/course-content/apps/api/src/app/app.module.ts

```diff
@@ -0,0 +1,31 @@
+import {Injectable, Module} from '@nestjs/common';
+import { QuestionManagementModule } from './question-management/question-management.module';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import {ConfigModule, ConfigService} from "@nestjs/config";
+import {UsersModule} from "./users/users.module";
```

Comment:
**suggestion:** always format your code. In WebStorm go to File/Settings=>search prettier=>set it to reformat on save and the other option.

After wards select all of your files under apps/api and do a Ctrl+Alt+L to format them and commit it back